# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/julia/no-remote-docs.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- julia-1.11.2/doc/make.jl.vanilla	2024-12-21 11:58:40.990000219 +0100
+++ julia-1.11.2/doc/make.jl	2024-12-21 11:59:12.723333558 +0100
@@ -376,7 +376,7 @@
     sitename  = "The Julia Language",
     authors   = "The Julia Project",
     pages     = PAGES,
-    remotes   = documenter_stdlib_remotes,
+    remotes   = nothing,
 )
 
 # Update URLs to external stdlibs (JuliaLang/julia#43199)
